digdag-standards/src/main/java/io/digdag/standards/operator/gcp/BqLoadOperatorFactory.java

```diff
@@ -122,9 +124,13 @@ private TableSchema tableSchema(Config params)
 
             String fileName = params.get("schema", String.class);
             try {
-                String schemaYaml = workspace.templateFile(templateEngine, fileName, UTF_8, params);
-                ObjectNode schemaJson = new YamlLoader().loadString(schemaYaml);
-                return objectMapper.readValue(schemaJson.traverse(), TableSchema.class);
+                String schemaString = workspace.templateFile(templateEngine, fileName, UTF_8, params);
+                if (FilenameUtils.getExtension(fileName).equals("json")) {
+                    return objectMapper.readValue(schemaString, TableSchema.class);
+                } else {
```

Comment:
The codes in this project follow https://github.com/airlift/codestyle. Could you put this `else` on a new line?